src/main/java/org/folio/circulation/domain/OverdueFineCalculatorService.java

```diff
@@ -217,7 +219,10 @@ private OverdueFineCalculatorService(Clients clients) {
       .thenCompose(r -> r.after(this::lookupItemRelatedRecords))
       .thenCompose(r -> r.after(this::lookupFeeFineOwner))
       .thenCompose(r -> r.after(params -> this.lookupLoggedInUser(params, loggedInUserId)))
-      .thenCompose(r -> r.after(params -> createAccount(fineAmount, params)));
+      .thenCompose(r -> r.after(params -> createAccount(fineAmount, params)))
+      .thenCompose(r -> r.after(feeFineAction ->
+        repos.scheduledNoticesRepository.deleteOverdueNotices(loan.getId())
```

Comment:
Could you please make sure that API endpoint permission's up to date for this new action?

Reply:
Added this permission to the three endpoints that can potentially do an overdue fine charge. Thanks for spotting this.